device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers_defaults_t1.j2

```diff
@@ -83,6 +83,9 @@
         },
         "{{ port_names }}|0-1": {
             "profile" : "[BUFFER_PROFILE|q_lossy_profile]"
+        },
+        "{{ port_names }}|5": {
+            "profile" : "[BUFFER_PROFILE|q_lossy_profile]"
```

Comment:
@andriymoroz-mlnx Is it the correct change to link queue 5 to lossy profile? pg 5 is handled by buffers_config.j2?

Reply:
yes, the code which handles these tables is common for all platforms
to be completely sure, I will test new buffers_config on one of the devices and confirm...

Reply:
it applies fine on MLNX